src/Elcodi/Component/Plugin/Repository/PluginRepository.php

```diff
@@ -15,13 +15,16 @@
  * @author Elcodi Team <tech@elcodi.com>
  */
 
-namespace Elcodi\Component\Menu\Event;
+namespace Elcodi\Component\Plugin\Repository;
 
-use Elcodi\Component\Menu\Event\Abstracts\AbstractMenuEvent;
+use Doctrine\ORM\EntityRepository;
 
 /**
- * Class PostMenuLoadEvent
+ * Class PluginRepository
  */
-class PostMenuLoadEvent extends AbstractMenuEvent
+class PluginRepository extends EntityRepository
 {
+    /**
```

Comment:
They don't apply anymore, maybe they were related to a former function that no longer exists.